### DISCUSSION TITLE
Problem during recreating same connectivity from MRTrix on DIPY

### DISCUSSION THREAD

**Discussion by legendaddy4552** [dipy/dipy#3226] [Tractography / Tractometry]
For a research project, We are trying to recreate the same connectivity matrix that is generated from Mrtrix (see the command line code below, for specific options) in Python. We extract coreg_parcellation file and tck file from Mrtrix and the use the relevant functions in nilearn and dipy packages in Python (see the full code attached). But we are not able to get the same connectome as is outputted by Mrtrix. Do you have any thoughts on what we might be missing?

[code2pdf_6644dafb32ad4 (1).pdf](https://github.com/dipy/dipy/files/15354050/code2pdf_6644dafb32ad4.1.pdf)

**Comment by skoudoro** [comment]
Hi @legendaddy4552,

I am sorry, it will take time to look in details at all this code. There are many packages used (some that I do not know like `gurobipy`and  I saw that DIPY is just reading the tractogram. 

I recommend to ask this question on Neurostar forum (https://neurostars.org/)

Maybe I did not understand what you are trying to achieve and I will be more helpful if it was fully DIPY or a more precise question.

Maybe someone else will have a beter idea.

**Comment by legendaddy4552** [comment]
Code is partially taken from here = https://github.com/dipy/dipy/blob/master/dipy/io/streamline.py#L91

Our main aim is being able to load tractography and connectivity of MRTrix to Python, because our optimization algorithms are going to work on python.

But unfortunately .csv file (created with MRtrix by tck2connectome function) does not match with tractography which is loaded and then assigned to parcels of patient.
